### ISSUE TITLE
2 TOTP services request

### ISSUE THREAD

**Issue by hooddy** [tiuub/KeeOtp2#51]
How to use TOTP for services required 2 TOTP? Like Huawei. It's request 2 codes.

![Безымянный](https://github.com/tiuub/KeeOtp2/assets/6899176/85c9e1e6-9e1c-4a05-826e-37ca391db39e)

**Comment by EvEvans** [comment]
I have not encountered a request of 2 TOTP codes before. Based on the wording in your screenshot "obtain two consecutive MFA codes", doesn't it simply require 2 consecutive codes? In which case you need to give the first code for 'code 1', and wait for a few seconds for the next code to appear, and then use the new code for 'code 2'.

If this is true, there is no need to update this plugin for this use-case I would say. Please confirm if this is the case or not.

**Comment by tiuub** [comment]
Indeed, I think it only requires a second code a few seconds after the first one. You could use the "Show TOTP Code"-window for that purpose. It shows the current code and the next code for the next interval above the current one. 

Therefore there is no need for an update. 

Kind regards
tiuub
